src/Tribe/REST/V1/Endpoints/Swagger_Documentation.php

```diff
@@ -123,7 +123,7 @@ protected function get_own_documentation() {
 			'get' => array(
 				'responses' => array(
 					'200' => array(
-						'description' => __( 'Returns the documentation for The Events Calendar REST API in Swagger consumable format.', 'the-event-calendar' )
+						'description' => __( 'Returns the documentation for The Events Calendar REST API in Swagger consumable format.', 'the-events-calendar' )
```

Comment:
* :no_entry_sign: Each line in an array declaration must end in a comma
* :no_entry_sign: Each array item in a multi-line array declaration must end in a comma

<sub>hash: 5749399ec519a5a695b71476ffe14b2f</sub>